styleguide/source/_patterns/03-organisms/article-footer.twig

```diff
@@ -0,0 +1,28 @@
+<footer class="joe__article-footer">
+  {% include "@molecules/references.twig" %}
+  
+  <div class="joe__article-footer--secondary">
+    {% if editor_note %}
+      {% include "@atoms/title-label.twig" with { "title_label": {"level" : "3", "text" : "Editor's Note" } } %}
```

Comment:
Would it make sense to loop through a footer object that had title, and text instead of separate variables and more if statements?

Reply:
I kind of wanted to capture the different fields that would be available here. It would be easier to just do a loop I think but doesn't really capture the different information that is available to display down here.